packages/cloud-cognitive/src/components/EditUpdateCards/_edit-update-cards.scss

```diff
@@ -0,0 +1,97 @@
+//
+// Copyright IBM Corp. 2022, 2022
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// Standard imports.
+@import '../../global/styles/project-settings';
+
+// Other Carbon settings.
+// TODO: @import 'carbon-components/scss/globals/grid/grid'; if needed
+
+// EditUpdateCards uses the following Carbon components:
+// TODO: @import(s) of Carbon component styles used by EditUpdateCards
+
+// EditUpdateCards uses the following Carbon for IBM Products components:
+// TODO: @import(s) of IBM Products component styles used by EditUpdateCards
+
+// Define all component styles in a mixin which is then exported using
+// the Carbon import-once mechanism.
+@mixin edit-update-cards {
+  $block-class: #{$pkg-prefix}--edit-update-cards;
+
+  .#{$block-class} {
```

Comment:
i don't think any of this is necessary. just out of curiosity was this part of the design spec? but also i think in this case we can let the consumer style the body of the cards to fit their needs.

Reply:
specifically regarding the textarea and input line styling. sorry if that wasn't clear

Reply:
I've moved those styles to `storybook-style.scss` so the consumer can style the body as their needs.